pulp_python/app/tasks/sync.py

```diff
@@ -82,19 +83,37 @@ def __init__(self, remote):
             out_q (asyncio.Queue): The out_q to send `DeclarativeContent` objects to.
 
         """
-        project_specifiers = ProjectSpecifier.objects.filter(remote=self.remote)
+        ps = ProjectSpecifier.objects.filter(remote=self.remote)
```

Comment:
I am uncomfortable with using the name ps so close to the name pb in the code.  The short names are convenient for line length, but could get mixed up making it less readable which doesn't feel as pythonic as a descriptive name here.

Reply:
That's reasonable.  The reason it was changed wasn't for brevity though, it's because there are other places below that use the variable name  ```project_specifiers``` and I didn't want them to collide.  So we'd have to come up with another name to use here.